db/migrations/20240806102534_alter_article_spam.js

```diff
@@ -0,0 +1,21 @@
+const table = 'article'
+
+exports.up = async (knex) => {
+  await knex.schema.table(table, (t) => {
+    t.float('spam_score').nullable()
+    t.boolean('is_spam').nullable()
+
+    t.index('spam_score')
```

Comment:
do we need a timestamp to record the time that run the spam check, if any further usage?

Reply:
I log the check time in lambda logging by now
https://github.com/thematters/spam-detection-serverless/blob/main/app/app.py#L29-L31

Reply:
add more logging https://github.com/thematters/matters-server/pull/4117/commits/744abf95f16113d712244c85bb83c3bc76b16f3f